prov/efa/src/rxr/rxr_ep.c

```diff
@@ -72,13 +72,18 @@ struct rxr_rx_entry *rxr_ep_rx_entry_init(struct rxr_ep *ep,
 	memset(&rx_entry->cq_entry, 0, sizeof(rx_entry->cq_entry));
 
 	/* Handle case where we're allocating an unexpected rx_entry */
-	if (iov) {
-		memcpy(rx_entry->iov, iov, sizeof(*rx_entry->iov) * iov_count);
-		rx_entry->cq_entry.len = ofi_total_iov_len(iov, iov_count);
-		rx_entry->cq_entry.buf = iov[0].iov_base;
+	if (msg->msg_iov) {
+		memcpy(rx_entry->iov, msg->msg_iov, sizeof(*rx_entry->iov) * msg->iov_count);
+		rx_entry->cq_entry.len = ofi_total_iov_len(msg->msg_iov, msg->iov_count);
+		rx_entry->cq_entry.buf = msg->msg_iov[0].iov_base;
 	}
 
-	rx_entry->cq_entry.op_context = context;
+	if (msg->desc)
+		memcpy(&rx_entry->desc[0], msg->desc, sizeof(*msg->desc) * msg->iov_count);
+	else
+		memcpy(&rx_entry->desc[0], 0, sizeof(*msg->desc) * msg->iov_count);
```

Comment:
Use memset instead of memcpy

Reply:
Will do. That was a lazy copy-paste on my part.